server/manager/src/main/java/org/apache/accumulo/manager/TabletGroupWatcher.java

```diff
@@ -1386,7 +1386,7 @@ private void handleDeadTablets(TabletLists tLists, WalStateManager wals)
           deadTablets.subList(0, maxServersToShow));
       Manager.log.debug("logs for dead servers: {}", deadLogs);
       if (canSuspendTablets()) {
-        store.suspend(deadTablets, deadLogs, manager.getSteadyTime());
+        store.suspend(deadTablets, deadLogs, manager.getSteadyTime().getMillis());
```

Comment:
I started working on this and made the changes and it looks like there will be a decent number. Here is the work in progress commit: https://github.com/apache/accumulo/commit/a4bc8f55dac0644de15b9f80db67dec8a9e370f7

I think it should be a follow on because it touches a lot of files and may introduce more merge conflicts when merging from 3.1 to elasticity so probably best to isolate it. I can open a follow on PR with that branch and change once this is merged.